### PR TITLE
fixed README: where to put 'application.yml'

### PR DESCRIPTION
**Pull request by Thrillberg**
I noticed that the README says to put 'application.yml' into the root directory but I believe it's usually in the config directory. That is also where '.gitignore' is looking. Is this correct?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ This is a rails app that can easily be [deployed with heroku](https://devcenter.
 
 ### Environment variables
 
-This app uses the [figaro](https://github.com/laserlemon/figaro) gem, so it is recommended that you place all your environment variables in a `application.yml` file in the root directory of the app. The **required** environment variables are:
+This app uses the [figaro](https://github.com/laserlemon/figaro) gem, so it is recommended that you place all your environment variables in a `application.yml` file in the config directory of the app. The **required** environment variables are:
```
Comment (dblock) [inline README.md]: Since it's a special directory, just like the `application.yml` I would make this `config` (quoted).

**Comment by ilyakava** [comment]
I'll leave this 100% up to @dblock, but I foresee some commit squashing needed

**Comment by dblock** [comment]
This looks good. Please squash your commits.

**Comment by ilyakava** [comment]
@Thrillberg http://stackoverflow.com/questions/5189560/squash-my-last-x-commits-together-using-git

**Comment by Thrillberg** [comment]
Okay, I think I figured it out.

**Comment by dblock** [comment]
Look good, merging.

**Pull request merged**